x-pack/plugins/security_solution/server/usage/detections/rules/get_metrics.mocks.ts

```diff
@@ -97,3 +98,2908 @@ export const getEmptySavedObjectResponse = (): SavedObjectsFindResponse<never, n
+        },
+        gapCount: {
+          value: 6,
+        },
+        avgTotalIndexDuration: {
+          value: 2610.1356466876973,
+        },
+        minTotalIndexDuration: {
+          value: 0,
+        },
+        minGapDuration: {
+          value: 2811,
+        },
+        avgTotalSearchDuration: {
+          value: 23.42902208201893,
+        },
+        minTotalSearchDuration: {
+          value: 1,
+        },
+        maxGapDuration: {
+          value: 5651,
+        },
+      },
+      'siem.savedQueryRule': {
+        doc_count: 0,
+        maxTotalIndexDuration: {
+          value: null,
+        },
+        avgGapDuration: {
+          value: null,
+        },
+        maxTotalSearchDuration: {
+          value: null,
+        },
+        gapCount: {
+          value: 0,
+        },
+        avgTotalIndexDuration: {
+          value: null,
+        },
+        minTotalIndexDuration: {
+          value: null,
+        },
+        minGapDuration: {
+          value: null,
+        },
+        avgTotalSearchDuration: {
+          value: null,
+        },
+        minTotalSearchDuration: {
+          value: null,
+        },
+        maxGapDuration: {
+          value: null,
+        },
+      },
+      'siem.eqlRule': {
+        doc_count: 0,
+        maxTotalIndexDuration: {
+          value: null,
+        },
+        avgGapDuration: {
+          value: null,
+        },
+        maxTotalSearchDuration: {
+          value: null,
+        },
+        gapCount: {
+          value: 0,
+        },
+        avgTotalIndexDuration: {
+          value: null,
+        },
+        minTotalIndexDuration: {
+          value: null,
+        },
+        minGapDuration: {
+          value: null,
+        },
+        avgTotalSearchDuration: {
+          value: null,
+        },
+        minTotalSearchDuration: {
+          value: null,
+        },
+        maxGapDuration: {
+          value: null,
+        },
+      },
+      'siem.thresholdRule': {
+        doc_count: 0,
+        maxTotalIndexDuration: {
+          value: null,
+        },
+        avgGapDuration: {
+          value: null,
+        },
+        maxTotalSearchDuration: {
+          value: null,
+        },
+        gapCount: {
+          value: 0,
+        },
+        avgTotalIndexDuration: {
+          value: null,
+        },
+        minTotalIndexDuration: {
+          value: null,
+        },
+        minGapDuration: {
+          value: null,
+        },
+        avgTotalSearchDuration: {
+          value: null,
+        },
+        minTotalSearchDuration: {
+          value: null,
+        },
+        maxGapDuration: {
+          value: null,
+        },
+      },
+      'siem.mlRule': {
+        doc_count: 0,
+        maxTotalIndexDuration: {
+          value: null,
+        },
+        avgGapDuration: {
+          value: null,
+        },
+        maxTotalSearchDuration: {
+          value: null,
+        },
+        gapCount: {
+          value: 0,
+        },
```

Comment:
Note: diff here is that cardinality agg is going to return 0 (if missing), but the max/min/avg aggs will return `null` as providing `missing` here will mess with the actual value.